src/commands/ai/ask.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const { flags } = require('@oclif/command');
+const utils = require('../../util');
+
+class AiAskCommand extends BoxCommand {
+	async run() {
+		const { flags, args } = this.parse(AiAskCommand);
+		let options = {};
+        options.mode = flags.items.length > 1 ? 'multi_item_qa' : 'single_item_qa';
+
+        if (flags.prompt) {
+            options.prompt = flags.prompt;
+        }
+        if (flags.items) {
+            options.items = flags.items;
+        }
+
+        let answer = await this.client.ai.ask({
+            prompt: options.prompt,
+            items: options.items,
+            mode: options.mode
+        });
+        await this.output(answer);
+	}
+}
+
+AiAskCommand.description = 'Sends an AI request to supported LLMs and returns an answer';
+AiAskCommand.examples = ['box ai:ask --mode single_item_qa --items=id=12345,type=file --prompt "What is the status of this document?"'];
```

Comment:
just remove this `--mode signle_item_qa` from the example and regenerate the md file